install_config/upgrading/automated_upgrades.adoc

```diff
@@ -329,6 +329,91 @@ $ ansible-playbook -i <path/to/inventory/file> \
 See xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Manging
 Nodes] for more on node labels.
 
+
+[[upgrade-hooks]]
+=== Customizing Upgrades With Ansible Hooks
+
+Ansible allows for operators to execute custom tasks during specific operations
```

Comment:
'OpenShift Ansible...' Is that something we've ever said before? Without 'OpenShift' this to me reads like this is built in ansible functionality, but maybe that's only me.